docs/dev/reference/dca/config.md

```diff
@@ -33,23 +33,25 @@ $GLOBALS['TL_DCA']['tl_example']['config'] = [
+| dynamicPtable      | true/false (`bool`)               | Dynamically set the parent table like in `tl_content`.                                                                                          |
 | ctable             | Child tables (`array`)            | Name of the related child tables (table.id = ctable.pid).                                                                                       |
 | dataContainer      | Data Container (`string`)         | Table (database table), File (local configuration file) or Folder (file manager).                                                               |
-| closed             | true/false (`bool`)            | If true, you cannot add further records to the table.                                                                                           |
-| notEditable        | true/false (`bool`)            | If true, the table cannot be edited.                                                                                                            |
-| notDeletable       | true/false (`bool`)            | If true, records in the table cannot be deleted.                                                                                                |
-| notSortable        | true/false (`bool`)            | If true, records in the table cannot be sorted.                                                                                                 |
-| notCopyable        | true/false (`bool`)            | If true, records in the table cannot be duplicated.                                                                                             |
-| notCreatable       | true/false (`bool`)            | If true, records in the table cannot be created but can be duplicated.                                                                          |
-| switchToEdit       | true/false (`bool`)            | Activates the "save and edit" button when a new record is added (sorting mode 4 only).                                                          |
-| enableVersioning   | true/false (`bool`)            | If true, Contao saves the old version of a record when a new version is created.                                                                |
-| doNotCopyRecords   | true/false (`bool`)            | If true, Contao will not duplicate records of the current table when a record of its parent table is duplicated.                                |
-| doNotDeleteRecords | true/false (`bool`)            | If true, Contao will not delete records of the current table when a record of its parent table is deleted.                                      |
+| markAsCopy         | Field name (`string`)             | Mark the new record with "copy of" in this field name. Since Contao 4.6                                                                         |
```

Comment:
```suggestion
| markAsCopy         | Field name (`string`)             | Appends "(copy)" to this field when copying a record (Contao __4.6__ and up).                                                                         |
```

Reply:
Ich kann leider nicht direkt in deinen branch comitten, müsstest du selbst comitten ;)